p/themes/base-theme/template.css

```diff
@@ -604,7 +608,7 @@ input[type="search"] {
 	table-layout: fixed;
 }
 
-#stream {
+#stream:not(.alert-warn) {
 	background: inherit;
```

Comment:
This rule otherwise had priority in other themes overriding `.alert-warn  { background: XXX }`